projects/plugins/jetpack/_inc/client/lib/plans/constants.js

```diff
@@ -117,6 +120,7 @@ export const JETPACK_MONTHLY_PLANS = [
 	PLAN_JETPACK_SECURITY_T1_MONTHLY,
 	PLAN_JETPACK_SECURITY_T2_MONTHLY,
 	PLAN_JETPACK_COMPLETE_MONTHLY,
+	PLAN_JETPACK_COMPLETE_MONTHLY,
```

Comment:
This should be `PLAN_JETPACK_GROWTH_MONTHLY`, right?
```suggestion
	PLAN_JETPACK_GROWTH_MONTHLY,
```

Reply:
Yes it should 😅 thank you for catching that!